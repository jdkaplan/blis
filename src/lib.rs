pub mod bytecode;
pub mod compile;
pub mod parse;
pub mod runtime;
pub mod vm;

#[cfg(test)]
mod testing;
