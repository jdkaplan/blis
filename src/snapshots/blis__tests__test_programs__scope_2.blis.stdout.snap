---
source: src/lib.rs
description: "let x = 1;\n{\n    let x = x + 1;\n}\nprint(x);\n"
input_file: test_programs/scope_2.blis
---
1
